pom.xml

```diff
@@ -33,14 +33,12 @@
         and  acceptance-tests/runner/scripts/args.sh
     -->
     <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
-    <jenkins.baseline>2.426</jenkins.baseline>
+    <jenkins.baseline>2.479</jenkins.baseline>
     <jenkins.version>${jenkins.baseline}.3</jenkins.version>
     <javadoc.exec.goal>javadoc-no-fork</javadoc.exec.goal> <!-- stop initialize phase plugins executing twice -->
-    <byte-buddy.version>1.14.9</byte-buddy.version>
+    <byte-buddy.version>1.17.1</byte-buddy.version>
```

Comment:
Not strictly necessary for this PR, but updating to the latest version to make it easier to support Java 25+.